webapp/src/components/link_tooltip/link_tooltip.jsx

```diff
@@ -42,7 +47,7 @@ export const getInfoAboutLink = (href, hostname) => {
     return {};
 };
 
-export const LinkTooltip = ({href, connected, gitlabURL, show}) => {
+export const LinkTooltip = ({href, connected, connectedGitlabUrl, show}) => {
```

Comment:
Thoughts on converting this file to typescript in this PR? Or at least changing the file extension temporarily to see if any important issues arise